### PR TITLE
Update flexbox menu

### PR DESCRIPTION
**Pull request by Nicki-lombardo**

### CODE REVIEW

File: css-flexbox_navigation/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  padding: 0;
+}
+
+.header {
+  padding: 20px;
+  color: var(--black-light);
+  background-color: var(--grey-light);
+}
+
+.navigation__menu {
```
Comment (adrianricken) [inline css-flexbox_navigation/css/styles.css:42]: i think you don't need the "align-items" and the "*height" line,

**Review by adrianricken — COMMENTED** [review]
looks good and works, just a few lines too much

File: css-flexbox_navigation/css/styles.css (hunk 1 above)
Comment (Nicki-lombardo) [reply css-flexbox_navigation/css/styles.css:42]: Thanks!

**Comment by Nicki-lombardo** [comment]
Merged!

**Pull request merged**